docs/blog/2020-10-03-bootloader-fix.md

```diff
@@ -176,15 +176,13 @@ this to all of the `.dts` files for the boards that were affected by this issue.
 
 ```diff
         code_partition: partition@26000 {
-            label = "code_partition";
```

Comment:
As discussed in Discord, adding a small "Changes" or "Edits" list to any changed blog posts would be good for posterity whenever changing old posts.

Reply:
Added. Not sure I picked the best heading for it, so feel free to suggest a better one.